<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nom_pere_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  20px;
border-radius : 8px solid gray;</string>
   </property>
   <property name="text">
    <string>ENREGISTREMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>500</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  20px;</string>
   </property>
   <property name="text">
    <string>Informations Parentale</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="photo_label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid  rgb(184, 184, 184);
border-radius : 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images.qrc">:/newPrefix/assets/image/161085384_4028971917170348_1595544585493431467_n.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pays">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>560</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Nom de la Mère</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_annuler_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>650</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : red;
color: white;
border-radius : 2 px solid red;
border: none;
font-size :  20px;
padding : 3px;
</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Nom du Père</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Domicile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lieu">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lieu_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Lieu de Naissance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_envoi_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>650</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color :  rgb(85, 255, 127);
color : white;
border-radius : 15 px solid white;
border : none;
font-weigth :bold;
font-size :  20px;
</string>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Nationnalité</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenom">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Entrez votre nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  20px;</string>
   </property>
   <property name="text">
    <string>Informations Personnelles</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_photo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 20px;</string>
   </property>
   <property name="text">
    <string>Choisir une photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Entrez votre prénom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="domicile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom_mere_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>590</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="sexe_h">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Homme</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="sexe_f">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size :  16px;</string>
   </property>
   <property name="text">
    <string>Femme</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
